packages/dx-react-scheduler-material-ui/src/templates/common/dialog/overlay.jsx

```diff
@@ -8,6 +8,12 @@ const styles = {
   modal: {
     position: 'absolute!important',
   },
+  paper: {
+    zIndex: '1302!important',
+  },
+  root: {
+    zIndex: '1301!important',
```

Comment:
Can we move this z-index to `modal` class?

Reply:
No, we cannot. Because Material-UI's BackDrop is also `modal` and has to have `position: 'absolute'` css prop, as the Dialog does. But the BackDrop shouldn't have `zIndex`. In case we move `zIndex` to `modal` class, the Dialog will start flickering on opening because BackDrop will be of the same `zIndex` as the Dialog

Reply:
Yeah, I've missed this class in BackdropProps